app/concepts/matestack/ui/core/collection/content/next/next.haml

```diff
@@ -0,0 +1,3 @@
+%a{@tag_attributes, "@click": "next()"}
```

Comment:
Use button?

Reply:
because not everybody would like to use a button. using an `a` tag gives the matestack user more flexibility